### DISCUSSION TITLE
A question about your tool

### DISCUSSION THREAD

**Discussion by distance20** [humhue/DAOTools#6] [Q&A] [answered]
hello , thanks for your work 
my english is poor
May i ask you how to use this dao tool
I am not specialty about this ,didn't find any executable file
some mod seems the same problem as Qwinn's fixpack
like
https://www.nexusmods.com/dragonage/mods/336/?  Dialogue Tweak
https://www.nexusmods.com/dragonage/mods/928/?  ZDF Dialog Fix
https://www.nexusmods.com/dragonage/mods/1120/?  Morrigan Restoration Patch

if this is to hard  plz tell me 
I just use qwinn's fixpack

**Comment by humhue** [comment, marked as answer]
This is a module for Python 3, so you have to install that first, then, as written in the readme you should execute these commands:

    pip3 install construct
    git clone https://github.com/humhue/DAOTools.git
    python3 DAOTools old.erf patched.erf

The git clone command is needed to download this library, but you can also download it by github (just be sure to unzip the downloaded file if you do so).
The last command is used for executing the main program of the module, to do so you have to write after "python3", the folder of DAOTools, it might be something like "C:\Users\your username\Downloads\DAOTools", after that, you have to write the path to the .erf file you want to patch, and the path to the file you want to create. After executing these commands you have to rename the new file name to the name of the old one.
Then you have to pack the .erf file back in its directory.
So let's say you want to get the path of an .erf file of one of these mods, you have to unzip the .zip file (and .dazip too, if there's one), you'll be left with a plain folder, so, just search for .erf files inside it and patch them doing the last command "python3" with the parameters you need as previously explained.

Also, if you like this project, please consider starring it on the main page.

**Reply by distance20** [reply]
Thanks for your answer
I've given a star
